Python/Product/PythonTools/PythonTools/Repl/BasePythonReplEvaluator.cs

```diff
@@ -1783,6 +1785,19 @@ class MemberResults {
         }
 
 #if DEV14_OR_LATER
+        internal static Version VSInteractiveVersion {
+            get {
+                if (_vsInteractiveVersion == null) {
+                    _vsInteractiveVersion = typeof(IInteractiveWindow).Assembly
+                        .GetCustomAttributes(typeof(AssemblyInformationalVersionAttribute), false)
+                        .OfType<AssemblyInformationalVersionAttribute>()
+                        .Select(a => { Version v; return Version.TryParse(a.InformationalVersion, out v) ? v : null; })
+                        .FirstOrDefault(v => v != null) ?? new Version();

```

Comment:
Getting the first non null entry should be fine. Because we do the version comparison though, shouldn't we get the greatest version?  Though, I am not sure you can end up with more than one `AssemblyInformationalVersionAttribute` anyway.  If that is the case, really shouldn't we validate there is only one non-null entry?